src/com/oltpbenchmark/DBWorkload.java

```diff
@@ -333,7 +338,7 @@ public static void main(String[] args) throws Exception {
             initDebug.put("Driver", wrkld.getDBDriver());
             initDebug.put("URL", wrkld.getNodes());
             initDebug.put("Isolation", wrkld.getIsolationString());
-            initDebug.put("Scale Factor", wrkld.getScaleFactor());
+            initDebug.put("Scale Factor", wrkld.getNumWarehouses());
```

Comment:
Change to say "Num Warehouses" instead of "Scale Factor"

Reply:
I already did that.